### PR TITLE
Add PointerLock unadjustedMovement article

### LINKED ISSUE #4067: content: PointerLock unadjustedMovement [2020-11-14]
* [draft](https://docs.google.com/document/d/1Dzm1yHhM9oukHmv6wCqCtPdt9obMIyDP6KmZ3wdJDms/edit#)
* [thread](https://groups.google.com/a/google.com/g/webdev-content/c/lC8eLmdqJvM)
* next step: @jpmedley to review [PR](https://github.com/GoogleChrome/web.dev/pull/4098)

### PR DESCRIPTION
**Pull request by beaufortfrancois**
This PR adds the article dedicated to the PointerLock unadjustedMovement option.

Fixes #4067

Live preview: https://deploy-preview-4098--web-dev-staging.netlify.app/mouse-acceleration/

### CODE REVIEW

**Comment by beaufortfrancois** [comment]
Once intent to ship is sent, I'll update the TODO in this article **before** merging.

File: src/site/content/en/blog/mouse-acceleration/index.md [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+        // You can request again a regular pointer lock.
+        return myTargetElement.requestPointerLock();
+      }
+    });
+}
+```
+
+It is possible to toggle between accelerated and non-accelerated mouse movement
+data without releasing the pointer lock. Simply request the pointer lock again
+with the desired option.  If that request fails, the original lock will remain
```
Comment (jpmedley) [inline src/site/content/en/blog/mouse-acceleration/index.md:127]:
```suggestion
with the desired option. If that request fails, the original lock will remain
```

File: src/site/content/en/blog/mouse-acceleration/index.md [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+intact and the returned promise will reject. No pointer lock events will fire
+for a failed change request.
+
+### Browser support {: #browser-support }
+
+The Pointer Lock API is [well supported across browsers]. However Chromium-based
+browsers (e.g. Chrome, Edge) are the only ones to support disabling OS-level
```
Comment (jpmedley) [inline src/site/content/en/blog/mouse-acceleration/index.md:134]:
```suggestion
browsers (e.g. Chrome, Edge, etc.) are the only ones to support disabling OS-level
```

Comment (jpmedley) [reply src/site/content/en/blog/mouse-acceleration/index.md:134]: I want to acknowledge that there are other Chromium browsers. I almost added Samsung, but they're behind us, I believe.

Comment (jpmedley) [reply src/site/content/en/blog/mouse-acceleration/index.md:134]: Or, is it that Chrome and Edge are the only ones the we know for sure support this?

**Review by jpmedley — CHANGES_REQUESTED** [review]
Looking good! I found a few nits. I don't think any of this requires discussion. We should be able to get this merged today.

File: src/site/content/en/blog/mouse-acceleration/index.md [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+[first-party perspective] games, commonly first party shooters (FPS), disabling
+mouse acceleration results in a better gaming experience and higher accuracy
+according to professional gamers.
+
+<figure class="w-figure">
+  <img src="./mouse-properties-dialog.png" class="w-screenshot" alt="Screenshot of the pointer motion control in Windows 10 settings.">
+  <figcaption class="w-figcaption">Pointer motion control in Windows 10 settings.</figcaption>
+</figure>
+
+Starting in Chrome 88, web apps can now switch back and forth between
```
[the remark this answers is not in the document]
Comment (beaufortfrancois) [reply src/site/content/en/blog/mouse-acceleration/index.md]: Done

File: src/site/content/en/blog/mouse-acceleration/index.md [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+
+document.addEventListener("pointerlockerror", () => {
+  console.log("pointer lock error");
+});
+```
+
+### Disable mouse acceleration {: #disable-mouse-acceleration }
+
+Call `requestPointerLock()` with  `{ unadjustedMovement: true }` to disable
+OS-level adjustment for mouse acceleration, and get access to raw mouse input.
```
[the remark this answers is not in the document]
Comment (beaufortfrancois) [reply src/site/content/en/blog/mouse-acceleration/index.md]: Done

File: src/site/content/en/blog/mouse-acceleration/index.md [hunk 5]
```diff
@@ -0,0 +1,175 @@
...
+This way, mouse movements data from `mousemove` events won't include the mouse
```
[the remark this answers is not in the document]
Comment (beaufortfrancois) [reply src/site/content/en/blog/mouse-acceleration/index.md]: Done

File: src/site/content/en/blog/mouse-acceleration/index.md (hunk 1 above)
Comment (beaufortfrancois) [reply src/site/content/en/blog/mouse-acceleration/index.md]: Done

File: src/site/content/en/blog/mouse-acceleration/index.md (hunk 2 above)
Comment (beaufortfrancois) [reply src/site/content/en/blog/mouse-acceleration/index.md]: Samsung Internet is mobile while this is supported only on desktop.
Done

**Review by beaufortfrancois — COMMENTED** [review]
FYI I still need to update the last TODO (Intent to ship) before publishing it.

**Review by jpmedley — APPROVED** [review]

**Comment by beaufortfrancois** [comment]
PR has been re-opened at https://github.com/GoogleChrome/web.dev/pull/4107

**Pull request merged**